pom.xml

```diff
@@ -1002,7 +1002,7 @@
         </property>
       </activation>
       <properties>
-        <spark.version>2.0.1</spark.version>
+        <spark.version>2.1.0</spark.version>
```

Comment:
We should keep running test on Spark 2.0.
Would you mind adding a new profile for Spark 2.1, like #283?